psi4/src/psi4/occ/cepa_iterations.cc

```diff
@@ -55,38 +55,42 @@ void OCCWave::cepa_iterations() {
                  // DIIS
     DIISManager t2_diis;
     if (nooA + nooB != 1) {
-        psio_->open(PSIF_OCC_DPD, PSIO_OPEN_OLD);
```

Comment:
What's the rationale for moving the `psio` calls inside the if-clauses? This should always run.

Reply:
This may have been the case I consulted you on where former restricted=ondisk and unrestricted=incore got consolidated to both ondisk. So this reverts.